docker/setup.sh

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+
+# Ensure script is always ran in Mbin's root directory.
+cd "$(dirname "$0")/.."
+
+if [[ "$1" == "" || "$1" == "-h" || "$1" == "--help" ]]; then
+    cat << EOF
+Usage: ./docker/setup.sh MODE DOMAIN
+Automate your Mbin docker setup!
+
+MODE needs to be either "prod" or "dev".
+DOMAIN will set the correct domain related fields in the .env file. Use "localhost" if you are just testing locally.
+
+Examples:
+  ./docker/setup.sh prod mbin.domain.tld
+  ./docker/setup.sh dev localhost
+EOF
+  exit 0
+fi
+
+case $1 in
+    prod)
+        mode=prod
+        ;;
+    dev)
+        mode=dev
+        ;;
+    *)
+        echo "Invalid mode provided: $1"
+        echo "Must be either prod (recommended for most cases) or dev."
+        exit 1
+        ;;
+esac
+
+domain=$2
+if [[ -z $domain ]]; then
+  echo "DOMAIN must be provided. Use "localhost" if you are just testing locally."
```

Comment:
```suggestion
  echo "DOMAIN must be provided. Use \"localhost\" if you are just testing locally."
```

And now we do need to backlash these quotes :P  sorry about that